java/java.source/src/org/netbeans/modules/java/ui/FmtSpaces.java

```diff
@@ -336,10 +336,70 @@ public String toString() {
     }
     
     private static final class SpacesCategorySupport extends CategorySupport {
-
+        // copied from Bundle.properties for easier maintanance
+        static String sampleSpaces ="""
```

Comment:
we shouldn't inline message bundle values, I suppose this was only done for debugging purposes? Changes to this file should be dropped IMO.